README.md

```diff
@@ -218,6 +218,38 @@ lambda-cfn save dev
 
 We're happy you want to contribute! Check out [CONTRIBUTING.MD](CONTRIBUTING.MD) for more information.
 
+## Dispatch Integration
+
+You have to set `DispatchSnsArn` variable when deploying your patrol rule and create a message following
```

Comment:
"In order to use [`dispatch`](https://github.com/mapbox/dispatch), you have to set..."